apps/trading/components/liquidity-supplied/liquidity-supplied.tsx

```diff
@@ -0,0 +1,143 @@
+import { useCallback, useMemo, useState } from 'react';
+import {
+  addDecimalsFormatNumber,
+  formatNumberPercentage,
+  NetworkParams,
+  t,
+  useDataProvider,
+  useNetworkParams,
+} from '@vegaprotocol/react-helpers';
+import type { MarketDealTicket } from '@vegaprotocol/market-list';
+import type {
+  MarketData,
+  MarketDataUpdateFieldsFragment,
+  SingleMarketFieldsFragment,
+} from '@vegaprotocol/market-list';
+import { marketDataProvider, marketProvider } from '@vegaprotocol/market-list';
+import { HeaderStat } from '../header';
+import { Link, Tooltip } from '@vegaprotocol/ui-toolkit';
+import BigNumber from 'bignumber.js';
+import { useCheckLiquidityStatus } from '@vegaprotocol/liquidity';
+import { MarketDataGrid } from '@vegaprotocol/deal-ticket';
```

Comment:
I think MarketDataGrid component has nothing market specific. 
I moved it in react-helpers because it is used in multiple places and I called it DataGrid.